lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -75,28 +75,6 @@ void WasmBinaryReader::InitializeReader()
 {
     ValidateModuleHeader();
     m_readerState = READER_STATE_UNKNOWN;
-#if DBG_DUMP
```

Comment:
I don't know if you have any strong feeling for this trace. I found it more annoying than anything